client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/RemoteShuffleResource.java

```diff
@@ -23,15 +23,17 @@ public class RemoteShuffleResource implements ShuffleResource {
 
   private final String rssMetaServiceHost;
   private final int rssMetaServicePort;
-
+  private final long rssMetaServiceTimeStamp;
```

Comment:
it's better that rssMetaServiceTimeStamp is replaced to rssMetaServiceTimestamp ?